### PR TITLE
Add BD message

### PR DESCRIPTION
**Pull request by kevinreiss**

### CODE REVIEW

File: app/views/shared/_announcement.html.erb [hunk 1]
```diff
@@ -1,10 +1,11 @@
 <%# to remove messages altogether, set this to empty string %>
-<% message = '' %>
+<% message = 'New BorrowDirect platform to launch 12/13/2022.' %>
 <% if Orangelight.read_only_mode %>
   <div class="col-12 alert alert-warning announcement">
     <div class="container">
       <p>
-      <%= message %> <%= Orangelight.read_only_message %>
+      <%= link_to message, "https://library.princeton.edu/news/general/2022-12-02/borrowdirect-new-platform-launch-december-13-2022" %>
```
Comment (sandbergja) [inline app/views/shared/_announcement.html.erb:7]: The link isn't displaying for me on qa.  Shouldn't this `link_to` be in the `elsif` block below?

**Review by sandbergja — CHANGES_REQUESTED** [review]
Thanks, @kevinreiss !  Just one comment.

File: app/views/shared/_announcement.html.erb (hunk 1 above)
Comment (kevinreiss) [reply app/views/shared/_announcement.html.erb]: That what explain why it didn't display. 😄

**Review by sandbergja — APPROVED** [review]
Thanks, @kevinreiss !

**Pull request merged**